src/GovUk.Education.ExploreEducationStatistics.Admin/Services/UserManagementService.cs

```diff
@@ -110,15 +110,19 @@ public async Task<UserReleaseRoleViewModel> GetUserReleaseRole(Guid userId, Guid
         {
             return await _persistenceHelper
                 .CheckEntityExists<UserReleaseRole>(userReleaseRoleId)
-                .OnSuccessDo(async () =>
+                .OnSuccess(async () =>
                 {
-                    var entityToRemove =
-                        await _contentDbContext.UserReleaseRoles.FirstOrDefaultAsync(r =>
-                            r.Id == userReleaseRoleId && r.UserId == userId);
-                    _contentDbContext.Remove(entityToRemove);
-                    await _contentDbContext.SaveChangesAsync();
-                })
-                .OnSuccess(_ => true);
+                    var entity = await _contentDbContext.UserReleaseRoles
```

Comment:
It looks like userId is not needed here at all to do this delete (UserReleaseRole has it's own surrogate Id) and the entity has already been fetched by the persistence helper in `CheckEntityExists` so it's available to the OnSuccess as the parameter.

Reply:
Yes good spot. Done. I also removed an unused endpoint.